deploy/ArbitrumRinkebyDeploy.js

```diff
@@ -169,101 +143,166 @@ module.exports = async (hre) => {
         fee
     )
 
+    await execute(
+        "PoolFactory",
+        {
+            from: deployer,
+            log: true,
+        },
+        "setInvariantCheck",
+        invariantCheck.address
+    )
+
     const BTC_POOL_CODE = "BTC/USD"
     const ETH_POOL_CODE = "ETH/USD"
 
-    const updateInterval = 3600 // 60 mins
-    const frontRunningInterval = 300 // 5 mins
+    const mintingFee = ethers.utils.parseEther("0.015")
+    const burningFee = ethers.utils.parseEther("0.015")
+    // const updateInterval = 3600 // 60 mins
+    // const frontRunningInterval = 300 // 5 mins
+    const updateInterval = 300 // 5 mins
+    const frontRunningInterval = 2400 // 30 seconds
     const oneLeverage = 1
     const threeLeverage = 3
-    // deploy LeveragePool
-    // BTC-USD 1x
+
+    // deploy ETH SMA Oracle
+    const ethSmaOracleWrapper = await deploy("EthUsdSMAOracle", {
+        from: deployer,
+        log: true,
+        contract: "SMAOracle",
+        args: [
+            ethOracleWrapper.address, //Oracle Address
+            24, // number of periods
+            300, // Update interval
+            deployer, // deployer address
+        ],
+    })
+
+    // Poll so there is an initial price
+    await execute(
+        "EthUsdSMAOracle",
+        {
+            from: deployer,
+            log: true,
+        },
+        "poll"
+    )
+
+    // deploy BTC SMA Oracle
+    // Note: if SMA oracle is not 24 periods you can't deploy oracle and pool in one go. Will need to wait until SMA oracle has enough periods before deploying pool.
```

Comment:
Is this still true with the ramping period?

Reply:
yeah pretty sure this is no longer true, will also add L2Encoder